app/interactors/create_group.rb

```diff
@@ -1,2 +1,13 @@
 class CreateGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.group_params
+  end
+
+  def execute
+    context.group = context.user.groups.create(context.group_params)
+  end
```

Comment:
I think this works great. In theory, if we needed to reuse this interaction for an admin (so that they could create groups for any given user), they can call this interaction and pass in any user needed.